src/main/java/address/util/ScreenSize.java

```diff
@@ -0,0 +1,29 @@
+package address.util;
+
+import java.io.Serializable;
+
+/**
+ * A Serializable class that contains the screen width and height.
+ */
+public class ScreenSize implements Serializable {
```

Comment:
Is this screen size or window size?
In future, you might be adding more attributes to this class. e.g. x-position, y-position, width of panel, etc. May be want to give a more general name. e.g. GuiSettings